pkg/controller/download_controller.go

```diff
@@ -188,11 +188,12 @@ func (c *downloadController) processDownloadItem(key string) error {
 		Callbacks: leaderelection.LeaderCallbacks{
 			OnStartedLeading: func(ctx context.Context) {
 				// Current node got the lease process request.
+				log.Infof("Lock is acquired by current node - %s to process Download %s.", c.nodeName, req.Name)
 				processErr = c.processDownload(req)
 				cancel()
 			},
 			OnStoppedLeading: func() {
-				log.Infof("Processed Download.")
+				log.Infof("Processed Download :%s", req.Name)
```

Comment:
We don't need to explicitly log `req.Name` here as in L134 `log` was set up with `key` as a common field.